functional_data_generator/pubspec.yaml

```diff
@@ -5,10 +5,10 @@ description: >-
 homepage: https://github.com/spkersten/dart_functional_data
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
 
 dependencies:
-  analyzer: '>=2.0.0 <5.0.0'
+  analyzer: '>=5.13.0 <6.0.0'
   build: ^2.0.1
   collection: ^1.15.0
   functional_data: ^1.0.0
```

Comment:
To check, it is compatible with older versions that don't have the new parameters?

Reply:
Forced functional data to 1.0.0 and still works fine (of course I cannot specify the params in the example file anymore, but generation works properly generating copy methods and lenses, would have been a problem if lenses was being default to false).

The reason is that I am using `peek` ins the generator, which accepts the annotation not existing. However I need to deal with the nullability there.
If I were using `read` instead, then it would throw an exception.